openid-sharedsignals-framework-1_0.md

```diff
@@ -1819,7 +1817,7 @@ status:
 ~~~ http
 GET /ssf/status?stream_id=f67e39a0a4d34d56b3aa1bc4cff0069f HTTP/1.1
 Host: transmitter.example.com
-Authorization: Bearer zzzz
+Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
```

Comment:
putting an actual token in GitHub sometimes causes GitHub to give alerts about stored tokens. Are you sure this value won't cause that alert?

Reply:
@tulshi All other examples in the SSF spec use the value `eyJ0b2tlbiI6ImV4YW1wbGUifQo=` as an example access token. If you are concerned about it, all other example access tokens in the SSF spec need to be replaced with `zzzz`.